apps/getting-started-cloudflare-workers/.gitignore

```diff
@@ -0,0 +1,172 @@
+# Logs
```

Comment:
Could we reduce the ignore list to a core set of entries relevant to this project?